tracer/test/test-applications/security/Samples.Security.AspNetCore5.DatabaseIntegration/Controllers/HomeController.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.AspNetCore.Mvc;
```

Comment:
I don't fully get why we need a new sample page instead of reusing/expanding the old one. I am pretty sure that there is a string reason, though.

Reply:
Yeah, I don't get why either. It feels like this could be added to the `Samples.Security.AspNetCore5` one

Reply:
It needs to be a whole new sample, because it has dependencies databases run in docker. The way the integration tests currently work is samples are either run with docker dependencies or without them - you can build in both tests runs.

So if we want to have these endpoints to `Samples.Security.AspNetCore5` it would mean moving all the tests in that run with docker dependency. We could so that, but I don't think it's what we want as the with docker run is more flaky.

Reply:
Also, somewhat separate, but it would be preferable to _link_ the files into the sample (if we really need one) rather than copying. Every extra file slows down CI a small amount 😄 + it's more duplication to deal with etc